docs/src/topics/SimpleCellValues_literate.jl

```diff
@@ -0,0 +1,95 @@
+# We start by including `Ferrite` and `Test`, 
+# to allow us to verify our implementation.
+using Ferrite, Test
+
+# Define a simple version of the cell values object, which only supports
+# identity mapping of scalar interpolations, without embedding. 
+struct SimpleCellValues{T, dim} <: Ferrite.AbstractCellValues
+    N::Matrix{T}             # Precalculated shape values
+    dNdξ::Matrix{Vec{dim,T}} # Precalculated shape gradients in the reference domain
+    dNdx::Matrix{Vec{dim,T}} # Cache for shape gradients in the real domain
+    M::Matrix{T}             # Precalculated geometric shape values
+    dMdξ::Matrix{Vec{dim,T}} # Precalculated geometric shape gradients
+    weights::Vector{T}       # Given quadrature weights in the reference domain
+    detJdV::Vector{T}        # Cache for quadrature weights in the real domain
+end;
+
+# To make it easier to initiate this struct, we create a constructor function
+# with the same input as `CellValues`. However, we skip some consistency checking here. 
+function SimpleCellValues(qr::QuadratureRule, ip_fun::Interpolation, ip_geo::Interpolation, T=Float64)
```

Comment:
Not sure about including `T` in the example here in this way, because this is not the same input as `CellValues`. Maybe we should just make this 2 functions as in CellValues (minus the checks).

Reply:
Good point! Decided to just take `T` as the eltype of the weights in the quadrature rule. 
No point in making this general.